data/upgrades/title.json

```diff
@@ -546,7 +546,13 @@
         "title": "Assailer",
         "type": "Title",
         "slots": ["Title"],
+        "grants": [
+          { "type": "slot", "value": "gunner", "amount": 1 },
+          { "type": "stat", "value": "hull", "amount": 2 },
+          { "type": "stat", "value": "shields", "amount": -2 }
+        ],
         "ffg": 769,
+        "image": "https://infinitearenas.com/xw2browse/epic/assailer.PNG",
```

Comment:
Hmm these are very bad quality pictures :( I wonder if there are any better ones available. I can't seem to find any

Reply:
I agree and hope someone has better.

Reply:
This image now 404s anyway